genesyscloud/organization_authentication_settings/resource_genesyscloud_organization_authentication_settings_schema.go

```diff
@@ -0,0 +1,124 @@
+		`minimum_length`: {
+			Description: "The minimum character length for passwords",
+			Optional:    true,
+			Type:        schema.TypeInt,
+		},
+		`minimum_digits`: {
+			Description: "The minimum number of numerals (0-9) that must be included in passwords",
+			Optional:    true,
+			Type:        schema.TypeInt,
+		},
+		`minimum_letters`: {
+			Description: "The minimum number of characters required for passwords",
+			Optional:    true,
+			Type:        schema.TypeInt,
+		},
+		`minimum_upper`: {
+			Description: "The minimum number of upper case letters that must be included in passwords",
+			Optional:    true,
+			Type:        schema.TypeInt,
+		},
+		`minimum_lower`: {
+			Description: "The minimum number of lower case letters that must be included in passwords",
+			Optional:    true,
+			Type:        schema.TypeInt,
+		},
+		`minimum_specials`: {
+			Description: "The minimum number of special characters that must be included in passwords",
+			Optional:    true,
+			Type:        schema.TypeInt,
+		},
+		`minimum_age_seconds`: {
+			Description: "Minimum age of the password (in seconds) before it can be changed",
+			Optional:    true,
+			Type:        schema.TypeInt,
+		},
+		`expiration_days`: {
+			Description: "Length of time (in days) before a password must be changed",
+			Optional:    true,
+			Type:        schema.TypeInt,
+		},
+	},
+}
+
+// ResourceOrganizationAuthenticationSettings registers the genesyscloud_organization_authentication_settings resource with Terraform
+func ResourceOrganizationAuthenticationSettings() *schema.Resource {
+	return &schema.Resource{
+		Description: `Genesys Cloud organization authentication settings`,
+
+		CreateContext: gcloud.CreateWithPooledClient(createOrganizationAuthenticationSettings),
+		ReadContext:   gcloud.ReadWithPooledClient(readOrganizationAuthenticationSettings),
+		UpdateContext: gcloud.UpdateWithPooledClient(updateOrganizationAuthenticationSettings),
+		DeleteContext: gcloud.DeleteWithPooledClient(deleteOrganizationAuthenticationSettings),
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		SchemaVersion: 1,
+		Schema: map[string]*schema.Schema{
+			`multifactor_authentication_required`: {
+				Description: `Indicates whether multi-factor authentication is required.`,
+				Optional:    true,
+				Type:        schema.TypeBool,
+			},
+			`domain_allowlist_enabled`: {
+				Description: `Indicates whether the domain allowlist is enabled.`,
+				Optional:    true,
+				Type:        schema.TypeBool,
+			},
+			`domain_allowlist`: {
+				Description: `The list of domains that will be allowed to embed Genesys Cloud applications.`,
+				Optional:    true,
+				Type:        schema.TypeList,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+			},
+			`ip_address_allowlist`: {
+				Description: `The list of IP addresses that will be allowed to authenticate with Genesys Cloud.`,
+				Optional:    true,
```

Comment:
Maybe we should put a warning in the description for `ip_address_allowlist` ? Describing the problem we had with this field